File: samples/workinprogress/websockets/app/controllers/Global.scala
import play.api._

import actors._
import akka.actor._
import java.util.concurrent._

object MyGlobal extends GlobalSettings {
  
  override def onStart(app: Application) {
    Logger.info("Start HelloActor")
    HelloActor.ref.start()
    /*Scheduler.schedule(HelloActor.ref, 
      """Hello""".stripMargin
      , 0, 1, TimeUnit.SECONDS)*/
  }
  
  override def onStop(app: Application) {
    Logger.info("Stop HelloActor")
    HelloActor.ref.stop()
  }
  
}

File: documentation/manual/working/javaGuide/main/global/code/javaguide/global/startstop/Global.java
/*
 * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
 */
package javaguide.global.startstop;

//#global
import play.*;

public class Global extends GlobalSettings {

    public void onStart(Application app) {
        Logger.info("Application has started");
    }

    public void onStop(Application app) {
        Logger.info("Application shutdown...");
    }

}
//#global
